src/pages/Environments.js

```diff
@@ -129,17 +129,21 @@ const Environments = () => {
 
   const refreshData = Utils.withBusyState(setLoading, async () => {
     const creator = getUser().email
-    const [newRuntimes, newDisks, galaxyDisks, newApps] = await Promise.all([
+    const [newRuntimes, newDisks, galaxyDisks, cromwellDisks, newApps] = await Promise.all([
       Ajax(signal).Runtimes.list({ creator }),
       Ajax(signal).Disks.list({ creator }),
-      Ajax(signal).Disks.list({ creator, saturnApplication: 'galaxy' }),
+      Ajax(signal).Disks.list({ creator, saturnApplication: tools.galaxy.appType }),
+      Ajax(signal).Disks.list({ creator, saturnApplication: tools.cromwell.appType }),
       Ajax(signal).Apps.listWithoutProject({ creator })
     ])
     setRuntimes(newRuntimes)
     setDisks(newDisks)
-    setGalaxyDisks(galaxyDisks)
     setApps(newApps)
 
+    _.forEach(disk => disk.appType = tools.galaxy.appType)(galaxyDisks)
+    _.forEach(disk => disk.appType = tools.cromwell.appType)(cromwellDisks)
+    setAppDisks(_.concat(galaxyDisks, cromwellDisks))
```

Comment:
Can I sell you on a functional version? With potential replacement for multiple calls above?
```js
setAppDisks(_.mapKeys(key => key === 'saturnApplication' ? 'appType' : key, newDisks)
```
or if you don't like renaming the keys
```js
setAppDisks(_.map(disk => _.set('appType', disk.saturnApplication, disk), newDisks)
```

Reply:
Unfortunately I can't really follow the suggestions, but the disk information returned does not include saturnApplication as a field. I definitely would like to remove the multiple calls, but I think that requires more information to be returned from the server so that we can tell which app or runtime a disk was associated with.

Reply:
```suggestion
    setAppDisks(_.concat(
      _.map(_.set('appType', tools.galaxy.appType), galaxyDisks),
      _.map(_.set('appType', tools.cromwell.appType), cromwellDisks)
    ))
```

Reply:
This code is no longer needed.